salt/grains/core.py

```diff
@@ -399,6 +399,73 @@ def _sunos_cpudata():
     return grains
 
 
+def _aix_cpudata():
+    '''
+    Return CPU information for AIX systems
+    '''
+    # Provides:
+    #   cpuarch
+    #   num_cpus
+    #   cpu_model
+    #   cpu_flags
+    grains = {}
+    cmd = '/usr/sbin/prtconf'
+    data = ""
+    if salt.utils.path.which(cmd):  # Also verifies that cmd is executable
+        data += __salt__['cmd.run']('{0}'.format(cmd))
+        data += '\n'
```

Comment:
```python
data = __salt__['cmd.run']('{0}'.format(cmd)) + os.linesep
```

Reply:
We are talking UNIX here, \n usage is similar to other usage in the file for UNIX system, for example: Solaris.  However I shall update to use os.linesep.

Reply:
done